<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Plot</class>
 <widget class="QWidget" name="Plot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>350</y>
     <width>681</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="label_min_x_2">
      <property name="text">
       <string>min x:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="min_x_2">
      <property name="minimum">
       <number>-1000000</number>
      </property>
      <property name="maximum">
       <number>1000000</number>
      </property>
      <property name="value">
       <number>-10</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_max_x_2">
      <property name="text">
       <string>max x:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="max_x_2">
      <property name="minimum">
       <number>-1000000</number>
      </property>
      <property name="maximum">
       <number>1000000</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_change_diapason">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>530</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Поменять диапазон</string>
   </property>
  </widget>
  <widget class="QCustomPlot" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>741</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>440</y>
     <width>681</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QLabel" name="label_min_y_2">
      <property name="text">
       <string>min y:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="min_y_2">
      <property name="minimum">
       <number>-1000000</number>
      </property>
      <property name="maximum">
       <number>1000000</number>
      </property>
      <property name="value">
       <number>-10</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_max_y_2">
      <property name="text">
       <string>max x:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="max_y_2">
      <property name="minimum">
       <number>-1000000</number>
      </property>
      <property name="maximum">
       <number>1000000</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
